### PR TITLE
Yjdh-135: Preserve selected language through the suomifi login

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:
Application will remember the previously selected langauge when coming back from suomifi login. Not only finnish but other languages too. Includes a new browser test that checks that the feature works with each foreign language (sv and en)
## Issues :bug:
https://helsinkisolutionoffice.atlassian.net/browse/YJDH-135

### CODE REVIEW

File: frontend/shared/browser-tests/components/header.components.ts [hunk 1]
```diff
@@ -8,10 +8,25 @@ import {
...
+  sv: {
+    login: 'Logga in',
+    logout: 'Logga ut',
+    language: 'På svenska',
+    userInfo: (user?: User) => new RegExp(`Användare: ${user?.name ?? ''}`),
+  },
+  en: {
+    login: 'Login',
+    logout: 'Logout',
+    language: 'In english',
```
Comment (Nizar-Rahme) [inline frontend/shared/browser-tests/components/header.components.ts:27]:
```suggestion
    login: 'Log in',
    logout: 'Log out',
    language: 'In English',
```

Comment (Nizar-Rahme) [reply frontend/shared/browser-tests/components/header.components.ts:27]: I'm probably just nitpicking here 😅

**Review by Nizar-Rahme — APPROVED** [review]
I left one small comment, otherwise, LGTM!

File: frontend/shared/browser-tests/components/header.components.ts (hunk 1 above)
Comment (nenonja) [reply frontend/shared/browser-tests/components/header.components.ts:27]: picked up the changes to my latest pr. Anyway translations are now made by me and will be real ones later.

**Pull request merged**